registerStyle.css

```diff
@@ -62,70 +62,70 @@
 .nav-list-item:nth-child(2),
 .nav-list-item:nth-child(3),
 .nav-list-item:nth-child(4) {
-    border-right: 2px solid #fff;
+    border-right:   2px solid #fff;
 }
 
 .nav-list-link {
-    display: block;
-    color: #fff;
-    padding: 1px 20px;
-    text-transform: uppercase;
-    text-decoration: none;
-    font-size: 18px;
-    font-family: Arial, Helvetica, sans-serif;
-    font-weight: 600;
+    display:   block;
+    color:   #fff;
+    padding:   1px 20px;
+    text-transform:   uppercase;
+    text-decoration:   none;
+    font-size:   18px;
+    font-family:   Arial, Helvetica, sans-serif;
+    font-weight:   600;
 }
 
 
 /* Registration page starts*/
 
 .Registration-page {
-    background: url(Rectangle\ 31.png);
-    background-repeat: no-repeat;
-    background-size: cover;
+    background:   url(Rectangle\ 31.png);
+    background-repeat:  no-repeat;
+    background-size:   cover;
 }
 
 .main {
-    background-color: #375FED;
-    height: 601px;
-    opacity: 0.8;
-    width: 601px;
-    float: right;
+    background-color:   #375FED;
+    height:   601px;
+    opacity:   0.8;
+    width:   601px;
+    float:   right;
 }
 
 .getstarted>h2 {
-    font-size: 40px;
-    font-family: monospace;
+    font-size:    40px;
+    font-family:   monospace;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/BuildForSDGCohort2/Team-317-Doctor-Appointment/pullRequest?prid=6127702)